src/lib/utils/resource-loader.ts

```diff
@@ -59,6 +60,29 @@ const getCoreResources = (type: string): Array<string> => {
     return ids;
 };
 
+const getInstalledResources = (type: string): Array<string> => {
+    const installedType = `installed-${type}`;
+
+    if (resourceIds.has(installedType)) {
+        return resourceIds.get(installedType);
+    }
+
+    const resourcesFiles: Array<string> = globby.sync(`${INSTALLED_ROOT}/@sonarwhal/${type}-*/**/package.json`);
```

Comment:
@molant You are using `${type}-*/**/package.json` instead of `${type}-*/package.json` because of the case where there are multiple packages in the same repository? If so, maybe add a comment stating that? Thanks!

Reply:
It's the value we had since the beginning  IIRC. We don't support multiple rules per package yet so this patter might change.